Project.toml

```
@@ -13,5 +13,5 @@ Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
 [compat]
 ChainRulesCore = "0.9.1"
 Compat = "3"
-FiniteDifferences = "0.10"
+FiniteDifferences = "0.11"
```

Since this didn't require any code changes,
we can keep supporting FiniteDifferences 0.10 as well.
So

```suggestion
FiniteDifferences = "0.10, 0.11"
```